experimenter/experimenter/nimbus_ui/templates/nimbus_experiments/launch_controls_v2.html

```diff
@@ -0,0 +1,201 @@
+        {% if experiment.is_rollout %}
+          {% include "nimbus_experiments/approval_rejection_controls.html" with action_label="End Rollout" approval_url="nimbus-ui-approve-end-experiment" cancel_reject_url="nimbus-ui-cancel-end-experiment" experiment=experiment origin="progress_card" replaced_element="#"|add:experiment.slug|add:"-progress-card" %}
+
+        {% else %}
+          {% include "nimbus_experiments/approval_rejection_controls.html" with action_label="End Experiment" approval_url="nimbus-ui-approve-end-experiment" cancel_reject_url="nimbus-ui-cancel-end-experiment" experiment=experiment origin="progress_card" replaced_element="#"|add:experiment.slug|add:"-progress-card" %}
+
+        {% endif %}
+      {% endif %}
+    </form>
+  {% endif %}
+  {% if experiment.is_enrolling %}
+    <div class="card bg-info-subtle p-4 mb-0 rounded-4">
+      <div class="row align-items-stretch">
+        <div class="col-5 py-4 mt-0">
+          <h5>Enrollment in progress — don't forget to close it</h5>
+          <p class="text-muted mb-0">
+            Enrollment is when participants can join your experiment. You'll need to end it yourself once enough people have joined, or when the time you set for enrollment has passed — it won't close automatically.
+          </p>
+          {% if experiment.should_show_end_enrollment %}
+            <div class="mt-4">
+              <form>
+                {% csrf_token %}
+                {% if experiment.is_rollout_update_requested %}
+                  {% include "common/cancel_review_button.html" with cancel_reject_url="nimbus-ui-cancel-update-rollout" experiment=experiment origin="progress_card" replaced_element="#"|add:experiment.slug|add:"-progress-card" %}
+
+                {% elif experiment.is_enrollment_pause_requested %}
+                  {% include "common/cancel_review_button.html" with cancel_reject_url="nimbus-ui-cancel-end-enrollment" experiment=experiment origin="progress_card" replaced_element="#"|add:experiment.slug|add:"-progress-card" %}
+
+                {% elif experiment.is_end_experiment_requested %}
+                  {% include "common/cancel_review_button.html" with cancel_reject_url="nimbus-ui-cancel-end-experiment" experiment=experiment origin="progress_card" replaced_element="#"|add:experiment.slug|add:"-progress-card" %}
+
+                {% elif experiment.should_show_end_enrollment or experiment.should_show_end_experiment or experiment.should_show_end_rollout or experiment.should_show_rollout_request_update %}
+                  {% if experiment.should_show_end_enrollment %}
+                    <button type="button"
+                            hx-post="{% url 'nimbus-ui-live-to-end-enrollment' slug=experiment.slug %}"
+                            hx-select="#{{ experiment.slug }}-progress-card"
+                            hx-target="#{{ experiment.slug }}-progress-card"
+                            hx-vals='{"origin": "progress_card"}'
```

Comment:
We can appendd the query params ourselves:

```suggestion
                            hx-post="{% url 'nimbus-ui-live-to-end-enrollment' slug=experiment.slug %}?origin=progress_card"
                            hx-select="#{{ experiment.slug }}-progress-card"
                            hx-target="#{{ experiment.slug }}-progress-card"
```

Although I would use a different param -- e.g., `?fragment=progress_card` or `?view=progress_card` or just `?progress_card`